### DISCUSSION TITLE
ICH9 Sound?

### DISCUSSION THREAD

**Discussion by libertyspike138** [Coopydood/ultimate-macOS-KVM#87] [Hardware]
The script creates an HDA ICH9 sound device but there is no sound. Is it possible to get sound working?

**Comment by eversiege** [comment]
I've created issue #88 based on this suggestion. While I am not sure this could even be done, sounds like good problem to try and tap into.

**Comment by chefnaphtha** [comment]
use a virtio sound device. macos has builtin support
